homeworks/homework-1/src/components/NewsList.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="uk-container uk-margin-top">
+    <h1 class="uk-heading-xlarge uk-text-center">News List</h1>
+
+    <div class="news-items-wrapper">
+      <table class="uk-table uk-table-striped uk-table-medium uk-table-middle">
+        <thead>
+          <tr>
+            <th class="uk-table-shrink">Id</th>
+            <th class="uk-table-expand">Title</th>
+            <th class="uk-table-shrink">Votes</th>
+            <th colspan="3" class="uk-table-shrink"></th>
+          </tr>
+        </thead>
+        <tbody v-if="newsItems.length > 0">
```

Comment:
```suggestion
        <tbody v-if="newsItems.length">
```
I know it's ugly, but in JavaScript, zero `0` evaluates to `false`.